MovieApp/MovieApp/App/CoreUI/ImageEnum.swift

```diff
@@ -0,0 +1,23 @@
+import UIKit
+
+enum ImageEnum {
```

Comment:
You should avoid giving names that indicate the type.
This can be just `Image` or `LocalImage` or `BundleImage` but without the `Enum` part.

Reply:
[Renamed](https://github.com/lihter/Movie-App/pull/24/files#diff-11d4a7ed0b85606e452de0c8c4f161b24f5b725b4096b1fda0299f87db8a1e73R3) to `BundleImage`.